### PR TITLE
[MM-33796][MM-33780] Updating channel switcher css system console

### PR DESCRIPTION
**Pull request by asaadmahmood**
#### Summary
MM-33796 - Updating channel switcher css system console
**I have also updated the elevations to use the correct color value.**

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-33796
https://mattermost.atlassian.net/browse/MM-33780

#### Screenshots
Screenshot 2021-03-14 at 12 58 39 PM

### CODE REVIEW

**Comment by esethna** [comment]
@amyblais

File: sass/components/_channel-switcher.scss [hunk 1]
```diff
@@ -94,12 +94,17 @@
             position: absolute;
             width: calc(100% - 48px);
             margin: -28px 0 0 24px;
-            background: rgba(var(--center-channel-bg-rgb), 1);
+            background: #fff;
```
Comment (devinbinnie) [inline sass/components/_channel-switcher.scss:97]: We should be using the System Console theme variables here.
```suggestion
            background: rgba(var(--sys-center-channel-bg-rgb), 1);
```

File: sass/components/_channel-switcher.scss [hunk 2]
```diff
@@ -94,12 +94,17 @@
...
             box-shadow: $elevation-4;
             border-radius: 4px;
-            border: 1px solid rgba(var(--center-channel-color-rgb), 0.16);
+            border: 1px solid #ddd;
```
Comment (devinbinnie) [inline sass/components/_channel-switcher.scss:100]: Same here (not sure if the opacity is right here)
```suggestion
            border: 1px solid rgba(var(--sys-center-channel-color-rgb), 0.16);
```

**Review by devinbinnie — CHANGES_REQUESTED** [review]

**Comment by asaadmahmood** [comment]
Done @devinbinnie

**Review by nevyangelova — APPROVED** [review]

**Review by devinbinnie — APPROVED** [review]

**Comment by asaadmahmood** [comment]
@devinbinnie @nevyangelova Looks like the ticket also pointed out another issue in the system console, so fixing that as well.
We can get your reviews again when @michaelgamble has approved this.

**Comment by jgilliam17** [comment]
@asaadmahmood Search is not returning results on `Add people to channel` modal. Can you please take a look? Thanks.

**Comment by asaadmahmood** [comment]
@jgilliam17 Do you have users added?
I just added some users and see the results.
Plus, that part is not changed here, it's only visual css changes.

Screenshot 2021-03-16 at 6 57 20 PM

**Review by michaelgamble — APPROVED** [review]
Took a look with Assad, looks like the issues are resolved:
- input field focus state color looks correct
- Chip design looks correct
- pop-over adhering to the correct theme looks correct

**Comment by jgilliam17** [comment]
@asaadmahmood one small issue 
- `X` to cancel modal seems to be using one og the current theme variables
Screen Shot 2021-03-16 at 9 57 13 AM

**Comment by asaadmahmood** [comment]
Should be fixed now @jgilliam17

**Review by jgilliam17 — APPROVED** [review]
Thank you @asaadmahmood 
Tested, looks good to merge.
- Verified theme of the dropdown and close `x`, input field focus and user chip are per design.

**Comment by amyblais** [comment]
@devinbinnie Please help review as a priority.

**Review by devinbinnie — APPROVED** [review]

**Comment by amyblais** [comment]
@asaadmahmood @devinbinnie Can you help cherry-pick to ``release-5.33`` and ``cloud`` branches please as a priority?

**Pull request merged**